src/Geolocation/Geolocation.php

```diff
@@ -158,6 +224,16 @@ public function getIpAddress(): string
 		return Variables::getGeolocationIp();
 	}
 
+	/**
+	 * Get geolocation expiration time.
+	 *
+	 * @return int
+	 */
+	public function getGeolocationExpiration(): int
+	{
+		return Variables::getGeolocationExpiration();
+	}
```

Comment:
Why the wrapper?

Reply:
So you can extend it in your project class if necesery.